### PR TITLE
URL fix Service now v2

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [x] In Progress
- [ ] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/45668

### CODE REVIEW

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml [hunk 1]
```diff
@@ -2493,7 +2493,7 @@ script:
     - contextPath: ServiceNow.Tasks.State
       description: state of task
       type: Unknown
-  dockerimage: demisto/python3:3.9.9.25564
+  dockerimage: demisto/python3:3.10.1.25933
```
Comment (Shellyber) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml:2496]: Let's revert the docker change for the time being (until the issue will be fixed)

File: Packs/ServiceNow/ReleaseNotes/2_2_21.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ServiceNow v2
+- Fixed an issue where given url could not contain ```v2```.
+- Updated the Docker image to: *demisto/python3:3.10.1.25933*.
```
Comment (Shellyber) [inline Packs/ServiceNow/ReleaseNotes/2_2_21.md:5]:
```suggestion
```

**Comment by Shellyber** [comment]
@Itay4 Can you force?
Validation fails on docker image

**Comment by Itay4** [comment]
merging due to the comment above

**Pull request merged**